.classpath

```diff
@@ -1,24 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-        <classpathentry including="**/*.java" kind="src" output="build/test-classes" path="test"/>
-        <classpathentry including="**/*.java" kind="src" path="src"/>
-        <classpathentry including="**/*.java" kind="src" path="gen"/>
-        <classpathentry kind="lib" path="lib/ant.jar"/>
-        <classpathentry kind="lib" path="lib/ant-launcher.jar"/>
-        <classpathentry kind="lib" path="lib/args4j.jar"/>
-        <classpathentry kind="lib" path="lib/gson.jar"/>
-        <classpathentry kind="lib" path="lib/guava.jar"/>
-        <classpathentry kind="lib" path="lib/jsr305.jar"/>
-        <classpathentry kind="lib" path="lib/junit.jar"/>
-        <classpathentry kind="lib" path="lib/protobuf-java.jar"/>
-
-        <!--
-        You need to tell Eclipse to build the compiler jar first, and then
-        normal source warnings will work because it will pick up the artifact.
-
-        If anyone has any better ideas, I'd be glad to hear them.
-        -->
-
-        <classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-        <classpathentry kind="output" path="build/classes"/>
+	<classpathentry including="**/*.java|**/*.properties" kind="src" path="src"/>
```

Comment:
Yes, the new file is what Eclipse generates when you use its UI to configure things. Happy to change this back, but I think it's better to just go with what Eclipse does here. .classpath is IMHO not really a human written, humand read source file, so I think it's fair to with simply what works best with the tooling.